lib/amorail/entities/company.rb

```diff
@@ -8,26 +10,26 @@ def request_attributes
           contacts: {
             add: [
               {
-                name: self.name,
-                linked_leads_id: [self.linked_leads_id],
+                name: name,
+                linked_leads_id: [linked_leads_id],
                 type: 'contact',
-                company_name: self.company_name,
+                company_name: company_name,
                 custom_fields: [
                   {
-                    id: 1460597,
-                    values: [{value: self.address}]
+                    id: properties.company.address.id,
+                    values: [{value: address}]
                   },
                   {
-                    id: 1460589,
-                    values: [{value: self.phone, enum: 'WORK'}]
+                    id: properties.company.phone.id,
+                    values: [{value: phone, enum: 'WORK'}]
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.